### PR TITLE
Hides the Rating component

### PR DESCRIPTION
**Pull request by supernova-at**
## Description

This PR hides the `Rating` component from the product tiles on the Category page because it has not been updated to the current version of Venia look and feel.

A separate ticket (PWA-2512) will update the component and re-enable it.

## Related Issue

Closes PWA-2510.

### Verification Stakeholders

@dpatil-magento 

## Verification Steps

1. Load up an environment where products have ratings (`rating_summary` GQL field has a value)
2. Verify that the `Rating` component does not display on the product tile on the Category page

#### Is Browser/Device testing needed?

No

#### Any ad-hoc/edge case scenarios that need to be considered?

No

## Checklist

-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.

### CODE REVIEW

File: packages/venia-ui/lib/components/Gallery/item.js [hunk 1]
```diff
@@ -68,9 +68,11 @@ const GalleryItem = props => {
         </div>
     );
 
-    const ratingAverage = rating_summary ? (
-        <Rating rating={rating_summary} />
-    ) : null;
+    // Hide the Rating component until it is updated with the new look and feel (PWA-2512).
+    const ratingAverage = null;
```
Comment (jeremycharron) [inline packages/venia-ui/lib/components/Gallery/item.js:72]: Two errors from `yarn run lint` : 
```
  17:8   error  'Rating' is defined but never used                   no-unused-vars
  48:54  error  'rating_summary' is assigned a value but never used  no-unused-vars
```
Perhaps remove lines or add a linter disable line?

**Review by jeremycharron — CHANGES_REQUESTED** [review]
See comment below

File: packages/venia-ui/lib/components/Gallery/item.js (hunk 1 above)
Comment (supernova-at) [reply packages/venia-ui/lib/components/Gallery/item.js:74]: Since we're going to re-enable this soon I just added linter disable lines.

I also added "remove the linter disable lines" to the scope of PWA-2512.

**Review by jeremycharron — APPROVED** [review]
Approved. Only need a version label and all good!

**Comment by supernova-at** [comment]
Closed in favor of #3654.
